### ISSUE TITLE
[Request] Add Support For AdventureBackpacks

### ISSUE THREAD

**Issue by Deadmano** [AzumattDev/AzuEPI#3]
With the new backpack slot added for the Backpacks mod, would you consider also allowing the [AdventureBackpacks](https://thunderstore.io/c/valheim/p/Vapok/AdventureBackpacks/) mod to automatically use that slot as well?

**Comment by AzumattDev** [comment]
> With the new backpack slot added for the Backpacks mod, would you consider also allowing the [AdventureBackpacks](https://thunderstore.io/c/valheim/p/Vapok/AdventureBackpacks/) mod to automatically use that slot as well?

It's simply not that easy. 
1. My mod isn't adding the backpack slot. Backpacks by Smoothbrain uses my API to tell my mod to make it. It comes with their own custom validations to validate what goes in it.
2. The author of AdventureBackpacks would have to add it(or you privately/manually in code)...or give me an API that I can use to validate his items. Since I have done it on my end before...but his items do weird shit due to his armatures being fucked.

Example. Being able to equip it...but it goes to the hand.
![image](https://github.com/AzumattDev/AzuEPI/assets/80414405/00b1fc71-0aec-427b-8cbd-badbc1fba9fb)
![image](https://github.com/AzumattDev/AzuEPI/assets/80414405/490a0a85-e565-4071-a3f9-0f0572a918cc)

TLDR; No. Likely it will never happen since he is basically MIA.
